### DISCUSSION TITLE
Are you interested in an artist?

### DISCUSSION THREAD

**Discussion by Maxog3n** [titinko/utsu#70] [General]
Hi! I'm really invested in the UTSU project, and I saw you needed someone to make icons or other art for your software on the UTAU forums. I don't really have a means for you to pay other than steam gift cards, so I'm happy to do em for free. Sorry for just barging in with this proposal all of a sudden! If you're interested, you can ask for some art examples.

**Comment by titinko** [comment]
Thanks for your generous offer!

The full art situation for UTSU is as follows:
1. The [original logo](https://github.com/titinko/utsu/blob/master/images/Hootsu.png) was designed by a [user](https://utaforum.net/members/soursop-the-fruit.3867/) on UTAforum back in 2018--however they are busy these days so there is currently no one in charge of icons.
2. I'm in talks with a small group about getting their help rebranding/marketing UTSU in exchange for prioritizing certain features, in which case the icons/art would need to match the new brand.  However this is still up in the air and may take months or end up not happening.
3. In the meantime UTSU has no icons (;_;) and no solid plan for getting them.

TL;DR Would definitely be interested in icon art, but I might need to change the art again as early as a few months from now.  If you're interested in doing art even though it may be temporary, I'd be happy to compensate you with Steam gift cards or any other method of your choosing!

**Reply by Maxog3n** [reply]
That would be awesome!! Thank you so much for taking in my offer. How should we communicate so you can get the icons you want?

**Reply by titinko** [reply]
Looks like you're on Utaforum so I'll private message you there.
